main/res/values/strings.xml

```diff
@@ -393,7 +394,7 @@
     <string name="caches_sort_favorites">Favorites</string>
     <string name="caches_sort_favorites_ratio">Favorites [%]</string>
     <string name="caches_sort_name">Name</string>
-    <string name="caches_sort_geocode">Geo Code</string>
+    <string name="caches_sort_geocode">Geocode</string>
```

Comment:
Is it one word in english? I am not sure.

Reply:
Yes, it is.
See our conversation in the first PR at https://github.com/cgeo/cgeo/pull/11372#discussion_r680344380.

Reply:
Thanks, now I remember :)